packages/web/README.md

```diff
@@ -220,6 +193,7 @@ Example:
 See the [LICENSE](LICENSE.md) file for information.
 
 [configuring-load-path]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#configuring-load-path
+[feature-flags-docs]: ../../docs/contributtion/feature-flags.md
```

Comment:
we should use github url IMHO. Because they might only see/have the web package and then this URL won't make sense.